Source/Fuse.Maps/MapOverlay.uno

```diff
@@ -115,6 +115,16 @@ namespace Fuse.Controls
 		static Selector _centerLatitudeName = "CenterLatitude";
 		static Selector _centerLongitudeName = "CenterLongitude";
 		static Selector _radiusName = "Radius";
+		static int UID_POOL = 0;
+		internal int uid = UID_POOL++;
```

Comment:
Code-style: Internal fields should start with an upper-case letter (`Uid`). Or perhaps just `Id`?

(It's unclear what the `U` means and `Id/id` seems to be used elsewhere.)

Reply:
the logic here is that I borrowed it from the MapMarker, uid is used to identify which MapOverlay has been tapped